OrbitCore/SymbolHelper.h

```diff
@@ -5,27 +5,34 @@
 #ifndef SYMBOL_HELPER_H_
 #define SYMBOL_HELPER_H_
 
+#include <filesystem>
 #include <string>
 #include <vector>
 
 #include "OrbitBase/Result.h"
 #include "symbol.pb.h"
 
 class SymbolHelper {
+  using path = std::filesystem::path;
```

Comment:
using fs = std::filesystem;

Reply:
done. It does not quite work this way, but I used `namespace fs = std::filesystem`.